packages/nodes-base/nodes/Postgres/Postgres.node.ts

```diff
@@ -11,14 +11,14 @@ export class Postgres extends VersionedNodeType {
 			name: 'postgres',
 			icon: 'file:postgres.svg',
 			group: ['input'],
-			defaultVersion: 2.1,
+			defaultVersion: 3,
 			description: 'Get, add and update data in Postgres',
 		};
 
 		const nodeVersions: IVersionedNodeType['nodeVersions'] = {
 			1: new PostgresV1(baseDescription),
 			2: new PostgresV2(baseDescription),
-			2.1: new PostgresV2(baseDescription),
```

Comment:
if someone has a workflow with version `2.1`, won't it break now?

Reply:
That is true. We added minor versioning in the meantime so I need to update node spec on this branch to use it and include existing minor version.